tests/unit/TooltipTest.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {screen, render, fireEvent} from '@testing-library/react';
+import ContextMenuItem from '../../src/components/ContextMenuItem';
+
+const Localize = require('../../src/libs/Localize');
+
+test('Show text in tooltip', () => {
+    render(<ContextMenuItem
+        icon=""
+        text={Localize.translate('en', 'reportActionContextMenu.copyToClipboard')}
+        successIcon=""
+        successText={Localize.translate('en', 'reportActionContextMenu.copyToClipboard')}
+        onPress={() => { }}
+    />);
+    const element = screen.getByText(Localize.translate('en', 'reportActionContextMenu.copyToClipboard'));
```

Comment:
This test is incomplete. You need to:

1. First verify that the tooltip text _is not_ shown on the screen
1. Then mouse over the element
1. Then verify that the tooltip text _is_ shown on the screen.